app/src/main/java/com/github/se/travelpouch/ui/navigation/SwipePager.kt

```diff
@@ -69,6 +80,7 @@ fun SwipePager(
                     when (pagerState.currentPage) {
                       0 -> navigationActions.navigateTo(Screen.TRAVEL_LIST)
                       1 -> selectedScreen = 0
+                      2 -> selectedScreen = 0
```

Comment:
It is not a new feature, but I think that it could be nicer to always go back to the list of travels as all of these screens are now at the same level.

Reply:
I think it is more interesting to always go back to a sort of base value before going back to list of travels. But we can discuss it later. It is no big deal